README.md

```diff
@@ -76,22 +76,17 @@ This sections provides details on what you need install on your system in order
     nvm install 6.12.3
     ```
 
+- _(Recommended)_ Node Package Manager(we strongly recommend you to use npm@3.10.10).
+	```
+	npm install -g npm@3.10.10
+	```
```

Comment:
This shouldn't be necessary since it's the default provided when you install Node.js v6.12.3.